### PR TITLE
docs: add individual ide process

### PR DESCRIPTION
**Pull request by stefanicjuraj**
- added individual process for each IDE  (`/usage/ide#vs-code`, `/usage/ide#vs-code-browser`,...)

### CODE REVIEW

**Review by ivan-burazin — APPROVED** [review]

**Review by ivan-burazin — APPROVED** [review]

File: src/content/docs/usage/ide.mdx [hunk 1]
```diff
@@ -61,11 +54,299 @@ You can set the default IDE used by Daytona.
...
+
+Daytona allows you to connect to your Workspace using Visual Studio Code (VSCode) Browser IDE.
+
+1. Run the following command to set your default IDE to VSCode Browser:
+
+   ```shell
+   daytona ide
+   ```
+
+    Upon running this command, Daytona will display a list of available IDEs, allowing you to select `VS Code` as your default choice.
```
Comment (Tpuljak) [inline src/content/docs/usage/ide.mdx:98]: `VS Code - Browser`

File: src/content/docs/usage/ide.mdx [hunk 2]
```diff
@@ -61,11 +54,299 @@ You can set the default IDE used by Daytona.
...
+
+    ```text
+    Default IDE: WebStorm
+    ```
+
+    You can now use the `daytona code` command to automatically open the JetBrains WebStorm IDE already connected to your Workspace.
+
+## Terminal SSH
+
+Daytona allows you to connect to your Workspace using Terminal SSH.
```
Comment (Tpuljak) [inline src/content/docs/usage/ide.mdx:330]: `Daytona allows you to connect to your Workspace using Terminal SSH.` => `Daytona allows you to connect to your Workspace through SSH`.

File: src/content/docs/usage/ide.mdx [hunk 3]
```diff
@@ -61,11 +54,299 @@ You can set the default IDE used by Daytona.
...
+      | Terminal SSH
+    ```
+
+2. Select `Terminal SSH` from the list presented. Upon selecting, Daytona will set it as the default IDE and display a confirmation message.
+
+    ```text
+    Default IDE: Terminal SSH
+    ```
+
+    You can now use the `daytona code` command to automatically open the Terminal SSH already connected to your Workspace.
```
Comment (Tpuljak) [inline src/content/docs/usage/ide.mdx:352]: `You can now use the `daytona code` command to automatically open the Terminal SSH` => `... to start an SSH session`.

File: src/content/docs/usage/ide.mdx [hunk 4]
```diff
@@ -61,11 +54,299 @@ You can set the default IDE used by Daytona.
...
+
+2. Select `WebStorm` from the list presented. Upon selecting, Daytona will set it as the default IDE and display a confirmation message.
+
+    ```text
+    Default IDE: WebStorm
+    ```
+
+    You can now use the `daytona code` command to automatically open the JetBrains WebStorm IDE already connected to your Workspace.
+
+## Terminal SSH
```
Comment (Tpuljak) [inline src/content/docs/usage/ide.mdx:328]: Terminal SSH is a bit special so it doesn't follow the "template" from above

**Review by Tpuljak — CHANGES_REQUESTED** [review]

**Review by ivan-burazin — APPROVED** [review]

**Review by Tpuljak — APPROVED** [review]

**Pull request merged**